.docker/config.json

```diff
@@ -0,0 +1,8 @@
+{
+        "auths": {
+                        "https://index.docker.io/v1/": {
+                        "auth": "Y29kYWN5YnVpbGQ6SmRrY0x4NlNZdGR3WUtjMm5xdHQ5Vjhm",
```

Comment:
Is there any way to avoid putthing the key on this file and put it on environment variable, for instance?

Reply:
I'm not even sure if we need this file. I just copy pasted from the withtools repository.
maybe we don't need it

Reply:
Removed :white